signup/models.py

```diff
@@ -236,8 +236,8 @@ class Contact(models.Model):
         " followed by last name)"))
     nick_name = models.CharField(_("Nick name"), max_length=60, blank=True,
         help_text=_("Short casual name used to address the contact"))
-    picture = models.URLField(_("Profile picture"), max_length=300, blank=True,
-        help_text=_("Profile picture S3 URL"))
+    picture = models.FileField(_("Profile picture"), max_length=300, blank=True,
```

Comment:
An [`ImageField`](https://docs.djangoproject.com/en/1.11/ref/models/fields/#django.db.models.ImageField) might be more appropriate. On the other hand, I looked in the db after an upload and the field is set to a name, not an absolute URL. That's a problem. `ImageField` takes a `storage` argument for overriding default_storage  but that might not help in multi-tenant situation (#22).

Reply:
We store absolute URLs in the database because it simplifies everything from code to data management. That's important when we are using third-party resources which are not directly under the Django application.

Reply:
Is it likely that we'll have a 3rd party resource scenario in the near future?